config/constructCalib.py

```diff
@@ -0,0 +1 @@
+config.visitKeys = ['expId']  # noqa: F821
```

Comment:
Why does this live here and not in obs_lsst?

Reply:
Or conversely, if it's in obs_lsst, why is it necessary here?

Reply:
Let alone something called exposure ID being assigned to something called visit...